software/index.rst

```diff
@@ -321,6 +321,35 @@ The following should get you set up with a working conda environment (replacing
 
 This has some limitations such as not supporting large model support. If you require this you can try the instructions below, these provide an older version of PyTorch however.
 
+TensorFlow Quickstart
+~~~~~~~~~~~~~~~~~~~~~
+The following should get you set up with a working conda environment (replacing <project> with your project code):
+
+
+::
+    export DIR=/nobackup/projects/<project>/$USER
+    # rm -rf ~/.conda .condarc $DIR/miniconda # Uncomment if you want to remove old env
+    mkdir $DIR
+    pushd $DIR
+
+    wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-ppc64le.sh
+
+    sh Miniconda3-latest-Linux-ppc64le.sh -b -p $DIR/miniconda
+    source miniconda/bin/activate
+    conda update conda -y
+
+    conda config --prepend channels \
+            https://public.dhe.ibm.com/ibmdl/export/pub/software/server/ibm-ai/conda/
+
+    conda config --prepend channels \
+            https://opence.mit.edu
+
+    conda create --name opence tensorflow -y
+    conda activate opence
+
+
+Note: This conflicts with the PyTorch instructions above as they set the conda channel_priority to be strict which seems to cause issues when installing TensorFlow.
```

Comment:
Do we need to change the PyTorch instructions then?

Reply:
I don't know, the PyTorch instructions work for PyTorch so I would be hesitant to change them.